src/sentry/static/sentry/app/components/charts/visualMap.tsx

```diff
@@ -0,0 +1,6 @@
+import {EChartOption} from 'echarts';
+import 'echarts/lib/component/visualMap';
+
+export default function VisualMap(visualMap: EChartOption.VisualMap) {
+  return visualMap;
+}
```

Comment:
I think this file should be located in `app/components/charts/components/`

Reply:
👍  Lets move after a merge